gen/builders/generator.go

```diff
@@ -126,37 +169,121 @@ func (g *Generator) MessageVectorGroup(group string, vectors ...*MessageVectorGe
 	go func() {
 		defer g.wg.Done()
 
+		var tmpOutPath string
+		if g.OutputPath != "" {
+			p, err := ioutil.TempDir(os.TempDir(), group)
+			if err != nil {
+				log.Printf("failed to create temp output directory: %s", err)
+				return
+			}
+			defer func() {
+				if err := os.RemoveAll(p); err != nil {
+					log.Printf("failed to remove temp output directory: %s", err)
+				}
+			}()
+			tmpOutPath = p
+		}
+
 		var wg sync.WaitGroup
 		for _, item := range vectors {
 			if id := item.Metadata.ID; g.Filter != nil && !g.Filter.MatchString(id) {
-				log.Printf("skipping %s", id)
+				log.Printf("skipping %s: matches filter", id)
```

Comment:
Shouldn't this say: `inclusion filter not matched`?

Reply:
Yes, I realised and fixed this already here: https://github.com/filecoin-project/test-vectors/pull/38/commits/f9f7a40a53d441957df4cefd5e5a598bf5caf2cd